packages/@aws-cdk/aws-elasticloadbalancingv2/lib/alb/application-listener.ts

```diff
@@ -539,3 +560,19 @@ export interface AddApplicationTargetsProps extends AddRuleProps {
    */
   readonly healthCheck?: HealthCheck;
 }
+
+/**
+ * Properties for adding a fixed response to a listener
+ */
+export interface AddFixedResponseProps extends AddRuleProps {
+  /**
+   * The fixed response
+   */
+  readonly fixedResponse: FixedResponse
```

Comment:
These properties should be inlined. You can do that by going `extends AddRuleProps, FixedResponse` and silencing the linter error.

Reply:
Modulo this, looks good